root/static/scripts/edit/URLCleanup.js

```diff
@@ -358,14 +366,24 @@ const linkToVideoMsg = N_l(
 );
 
 /*
- * CLEANUPS entries have 2 to 4 of the following properties:
+ * CLEANUPS entries have 2 to 5 of the following properties:
  *
  * - match: Array of regexps to match a given URL with the entry.
  *          It is the only mandatory property.
  * - restrict: Array of possible relationship types for the matched URL.
- *          It contains at most 1 relationship type by entity type.
+ *          May contain 1 or more relationship type by entity type.
+ *          You can use multiple() to combine multiple types.
  *          Will be used in auto-selection and validation.
- *          If there's only one type, it will be auto-selected.
+ *          If there's only one type or type combination,
+ *          it will be auto-selected.
+ * - select: Custom function for auto-selection of relationship type(s).
+ *          To select a single type, return a string,
+ *          e.g: LINK_TYPES.otherdatabases.release.
+ *          To select multiple types, return an array of strings,
+ *          e.g: [
+ *            LINK_TYPES.otherdatabases.work,
+ *            LINK_TYPES.lyrics.work,
+ *          ].
```

Comment:
Please fix indent for this commit too as suggested in earlier comment to the first commit.